CHANGELOG.md

```diff
@@ -1,5 +1,52 @@
 # [Versions](https://mui.com/versions/)
 
+## 6.0.0-rc.0
+
+<!-- generated comparing v6.0.0-beta.6..next -->
+
+_Aug 22, 2024_
+
+A big thanks to the 12 contributors who made this release possible. Here are some highlights ✨:
+
+- ⚡ Rendering performance improvements
+
+### `@mui/material@6.0.0-rc.0`
+
+- [Box] Remove `component` from `BoxOwnProps` (#43384) @DiegoAndai
```

Comment:
This should be under Breaking changes section

Reply:
Updated https://github.com/mui/material-ui/pull/43388/files#diff-06572a96a58dc510037d5efa622f9bec8519bc1beab13c9f251e97e657a9d4edR15